andela/shadowcat-backend
# #167729793 Book-Return-Trip-Ticket

#### What does this PR do?
This endpoint API allows a User to book a return Trip
#### Description of Task to be completed?
- implemented validation for User input
- wrote tests for returnTrip
- added a new seed to the database for test
#### How should this be manually tested?
you should pull this branch from origin `git pull origin` and run `npm run test`
- Can be tested with postman by visiting these routes
- post /api/v1/trips/request
- required field: departureDate, returnDate, tripType, destination, currentOfficeLocation, accommodation, and reason
#### Any background context you want to provide?
N/A
#### What are the relevant pivotal tracker stories?
[Finishes#167729793]
#### Screenshots (if appropriate)
![returnTrip test](https://user-images.githubusercontent.com/43038045/64913736-87cc6c00-d6fa-11e9-885a-b0671d581377.JPG)
![ReturnTrip](https://user-images.githubusercontent.com/43038045/64913737-8dc24d00-d6fa-11e9-8ab8-3f8c9216aacc.JPG)

#### Questions:
N/A
